src/components/pages/Animations/AnimationMediaHelper/useMedia.js

```diff
@@ -14,6 +14,7 @@ const useMedia = query => {
     media.addEventListener(Animation, listener);
 
```

Comment:
Several use effects had the warning about unincluded items in the dependency array. I am not sure what this error is referencing or how to fix it while maintaining the functionality of the useEffect - if anyone has any clarification on that please let me know.

Reply:
Why not include 'matches' to the dependency array? (ie `[query, matches]`)